assignments/arrays.js

```diff
@@ -63,31 +63,33 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log({ "id": 33, "car_make": "Jeep", "car_model": "Wrangler", "car_year": 2011 },);
```

Comment:
There's a way to do this without hard coding it. 😄 

use the inventory object and dot notation to console log the needed items. I'll show you the first and you can do the rest.... 

```
console.log(`Car 33 is a ${inventory[32].car_year}....`);
```